### DISCUSSION TITLE
Question on log rotation?

### DISCUSSION THREAD

**Discussion by killmasta93** [pfelk/pfelk#528] [Q&A] [answered]
Hi 
I was wondering how does the log rotation work? is there a way to put x days to keep logs?

Thanks

**Comment by a3ilson** [comment, marked as answer]
This is managed by the Index Lifecycle Polices. pfelk utilizes [this](https://raw.githubusercontent.com/pfelk/pfelk/main/etc/pfelk/templates/pfelk-ilm) one which can be changed/revised as desired.

1. Alternatively, you may navigate to the Management>>Index Management>>Templates(select pfelk) and edit. From here you may remove the index settings which will allow you to edit individually from the "Index Management" page.
<img width="1511" alt="Screenshot 2024-02-25 at 12 44 52" src="https://github.com/pfelk/pfelk/assets/16884679/88010ea7-2c98-4cef-9c4b-216c3d22d79a">

2. Once/if ILM is remove from the pfelk index settings, you should then be able to define the data retention here
<img width="1511" alt="Screenshot 2024-02-25 at 12 46 55" src="https://github.com/pfelk/pfelk/assets/16884679/cc27bcd1-abae-4dba-a123-c01cd34704e2">

**Comment by killmasta93** [comment]
Thanks for the reply, tried to replicate but i get invalid json

![image](https://github.com/pfelk/pfelk/assets/13953629/8fb497ba-4243-498c-b18f-de9d33dd897f)
